### ISSUE TITLE
Duplicated call of onConnected() in Spring

### ISSUE THREAD

**Issue by renjithraj2005** [mrniko/netty-socketio#1015]
I'm trying to set up a Socket.IO server with Spring boot, which also serves as an API for my application. I have a React frontend that consumes socket

The problem is when I call my service to connect, it calls the onConnected method of my Spring back twice. I tried to simplify it by removing, for example, the call to the client.joinRoom(sessionCode) method but the logs remain the same.

**Comment by jakechen0816** [comment]
Yes, I had this same issue. In Java, `addConnectListener` is called twice when client connects to the server.
